CalendarFXView/src/main/java/com/calendarfx/view/DateControl.java

```diff
@@ -1038,6 +1037,114 @@ public String toString() {
         }
     }
 
+    /**
+     * Possible edit operations on an entry.
+     * This enum will be used as parameter of the callback set with {@link DateControl#setEntryEditPolicy}.
+     */
+    public enum EditOperation {
+
+        /**
+         * Checked if the start of an entry can be changed (e.g. using drag/drop).
+         */
+        CHANGE_START,
+
+        /**
+         * Checked if the end of an entry can be changed (e.g. using drag/drop).
+         */
+        CHANGE_END,
+
+        /**
+         * Checked if entry can be moved using drag/drop.
+         */
+        MOVE,
+
+        /**
+         * Checked if an entry can be deleted.
+         */
+        DELETE
+    }
+
+    /**
+     * Class used for parameter of {@link com.calendarfx.view.DateControl#entryEditPolicy} functional interface.
+     */
+    public static final class EntryEditParameter {
+
+        /**
+         * The date control the entity is associated with.
+         */
+        private final DateControl dateControl;
+
+        /**
+         * The entity the operation is operated on.
+         */
+        private final Entry<?> entry;
+
+        /**
+         * The operation.
+         */
+        private final DateControl.EditOperation editOperation;
+
+        public EntryEditParameter(DateControl dateControl, Entry<?> entry, DateControl.EditOperation editOperation) {
+            this.dateControl = dateControl;
+            this.entry = entry;
+            this.editOperation = editOperation;
+        }
+
+        /**
+         * The {@link DateControl} which is asking for a specific {@link com.calendarfx.view.DateControl.EditOperation} permission.
+         * @returns The date control.
+         */
+        public DateControl getDateControl() {
+            return dateControl;
+        }
+
+        /**
+         * The entry where the {@link com.calendarfx.view.DateControl.EditOperation} should be applied.
+         * @returns The entry.
+         */
+        public Entry<?> getEntry() {
+            return entry;
+        }
+
+        /**
+         * The actual edit operation.
+         * @returns The edit operation.
+         */
+        public DateControl.EditOperation getEditOperation() {
+            return editOperation;
+        }
+    }
+
+    /**
+     * If an action will be issued on an item the given instance will be asked if the action is allowed.
```

Comment:
"If an edit operation will be issues ...." Please also add an @see EditOperation